mod.cpp

```diff
@@ -1,13 +1,13 @@
 name = "Gruppe Adler APOBS";
-author = "DerZade, Salbei";
+author = "Salbei, DerZade";
```

Comment:
```suggestion
author = "DerZade, Salbei";
```
Das ist alphabetisch geordnet